front/app/containers/Admin/pagesAndMenu/containers/ContentBuilder/components/Widgets/CommunityMonitorCTA/index.tsx

```diff
@@ -0,0 +1,158 @@
+
+  const isSurveyLive = phase?.data.attributes.submission_enabled;
+
+  // Get the survey UI schema
+  const { uiSchema } = useInputSchema({
+    projectId: project?.data.id || '',
+    phaseId,
+  });
+
+  // Calculate estimated time to complete survey
+  const estimatedMinutesToComplete = calculateEstimatedSurveyTime(uiSchema);
+
+  const goToCommunityMonitorSurvey = () => {
+    if (phaseId) {
+      clHistory.push(
+        `/projects/${project.data.attributes.slug}/surveys/new?phase_id=${phaseId}`
+      );
+    }
+  };
+
+  if (!isSurveyLive) {
+    return null;
+  }
+
+  return (
+    <Box
+      mx="auto"
+      w="100%"
+      maxWidth="1200px"
+      px={isTabletOrSmaller ? '16px' : '0'}
+    >
+      <Box
+        background={theme.colors.tenantPrimaryLighten90}
+        w="100%"
+        borderRadius="16px"
+        p="32px"
+        display="flex"
+        flexDirection={isTabletOrSmaller ? 'column' : 'row'}
+        gap="16px"
+      >
+        <Box maxWidth={isTabletOrSmaller ? '100%' : '40%'}>
+          <Title mt="0px" color="grey800" variant="h2" fontWeight="semi-bold">
+            {localize(title)}
+          </Title>
+          <Text mb="0px">{localize(description)}</Text>
+        </Box>
+
+        <Box
+          my="auto"
+          ml={isTabletOrSmaller ? undefined : 'auto'}
+          mx={isMobileOrSmaller ? 'auto' : undefined}
+        >
+          <Image
+            src={SentimentQuestionPreview}
+            alt={formatMessage(messages.sentimentQuestionPreviewAltText)}
+          />
+        </Box>
+
+        <Box
+          my="auto"
+          ml={isTabletOrSmaller ? undefined : 'auto'}
+          mx={isMobileOrSmaller ? 'auto' : undefined}
+          width={isMobileOrSmaller ? '100%' : undefined}
+        >
+          <Box
+            display={isMobileOrSmaller ? undefined : 'flex'}
+            justifyContent={
+              isMobileOrSmaller || !isTabletOrSmaller ? 'center' : undefined
+            }
+          >
+            <Button onClick={goToCommunityMonitorSurvey}>
+              {localize(surveyButtonText)}
+            </Button>
+          </Box>
+          <Box
+            display="flex"
+            justifyContent={
+              isMobileOrSmaller || !isTabletOrSmaller ? 'center' : undefined
+            }
+            mt="8px"
+          >
+            <Icon
+              width="14px"
+              fill={theme.colors.tenantPrimary}
+              name="clock-circle"
+              mr="4px"
+            />
+            <Text color="tenantPrimary" m="0px" mt="2px" fontSize="s">
+              {formatMessage(messages.xMinutesToComplete, {
+                minutes: estimatedMinutesToComplete,
+              })}
+            </Text>
+          </Box>
+        </Box>
+      </Box>
+    </Box>
+  );
+};
+
+CommunityMonitorCTA.craft = {
+  related: {
+    settings: Settings,
+  },
+};
+
+export const CommunityMonitorCTATitle = messages.communityMonitorCTATitle;
```

Comment:
```suggestion
export const communityMonitorCTATitle = messages.communityMonitorCTATitle;
```

Just for consistency